.github/CI.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
```

Comment:
I think this needs to go under `/.github/workflows`, otherwise GitHub actions won't pick it up.

Reply:
Sorry about that that was definitely it. 
I see that it is now running on my fork 

<img width="1684" alt="Screen Shot 2022-08-23 at 2 22 34 AM" src="https://user-images.githubusercontent.com/8097956/186109757-7073d4ef-9e47-4d2b-835d-7d47b39647b8.png">